nixos/modules/services/networking/firewall.nix

```diff
@@ -38,6 +38,10 @@ let
 
   cfg = config.networking.firewall;
 
+  uniqueListOf = type: let
+    elemType = types.listOf type;
+  in elemType // { merge = loc: defs: unique (elemType.merge loc defs); };
```

Comment:
Does `uniq (listOf ...)` not work?

Reply:
`uniq` ensures that there is only one >location< in the entire configuration that sets the option. It does not enforce unique elements in the list.

Reply:
Ohh, I somehow thought you wanted the former, ignore this then